lab-davide/test/greet-test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
This is a good start but requires a small change since this test is causing Travis CI to fail.  Since your greet() returns hello + name + how are you?, the result should equal "hello davide how are you?" in your assertion on line 10, not "hello davide".